_1327/main/tests.py

```diff
@@ -1,11 +1,13 @@
 import datetime
+from io import StringIO
```

Comment:
could we clean the imports here? I think this line should be underneath the plain `import` lines, shouldn't it?

Reply:
when using
```
import datetime
import json
import re
from io import StringIO
```
the linter says:
```
_1327/main/tests.py:4:1: I100 Import statements are in the wrong order. 'from io import StringIO' should be before 'import re'
```